kivy/core/text/__init__.py

```diff
@@ -671,7 +715,7 @@ def _render_real(self):
         size = self.size
         valign = options['valign']
 
-        y = ypad = options['padding_y']  # pos in the texture
+        y = ypad = options['padding'][1]  # pos in the texture
```

Comment:
Same applies here, probably better to just use `options['padding'][1]` instead of a generic `ypad` variable.